app.js

```diff
@@ -177,6 +178,14 @@ History.prototype.toJSON = function () {
     return this.facts.reverse().toJSON();
 };
 
+function buildFinalTRpath(baseURL, thisVersionURL) {
+
+    var w3URL = /https?:\/\/(www\.)?w3\.org\/tr/i;
```

Comment:
:-1:

Regular expressions often obfuscate the code and here is a good example. Reading this function, I cannot tell what is its purpose. When ultra-optimization is not in order, readability should be the key. Could you provide a regex-free alternative?

In terms of conservatism for the replacement, `This Version` URLs are never under `https`, always provided with `www`, and case is important (everything lower-case except for `TR`, always uppercase) so I am not sure you need to take extra care here.